### ISSUE TITLE
view_column icon does not align with integer pixels

### ISSUE THREAD

**Issue by leosdad** [google/material-design-icons#1731]
Contrary to the older Material Icons, the filled Material Symbols icon does not align with integer pixels when rendered at 24 px width. The screenshot shows how it looks in Figma at 100%:

![image](https://github.com/google/material-design-icons/assets/7026091/8da1509c-e973-4435-a7dc-23035acb2cec)

A zoomed view shows the misalignment:

![image](https://github.com/google/material-design-icons/assets/7026091/d2141d2a-f0e1-4cba-9554-5adfe11fbad4)

**Comment by tphinney** [comment]
tl;dr: as designed

So, yes, this sparked LENGTHY discussions when we were first working on the Material Symbols font a couple years ago. (Note to other viewers: the icon presented is the “Sharp” version, but the problem exists regardless.)

You did not draw the 24x24 box around the old Material Icons version. If you had, you might have noticed that… it is not centered in its box! So it has a problem, it is just a _different_ problem.

There are a lot of tradeoffs. If the main design had 2 px outside margins, 1 px between columns, with columns 6 px wide, that would have worked, but the default (unfilled) version would have been a box shape that was wider than standard—which would have been my preferred compromise choice, but not the designer’s.

**Comment by leosdad** [comment]
> tl;dr: as designed
> So, yes, this sparked LENGTHY discussions...

Fully understood. Yes, the old icon is misaligned. I just tried your description and I suppose I'd vote for it too. IMHO a better solution would be this:

![image](https://github.com/google/material-design-icons/assets/7026091/d87e821f-ba4a-48c8-b48b-08ff71dbc687)

i.e. 3 px wide bars. At least for the filled version it would work better. It's fully simmetrical and 100% on the grid.

**Comment by leosdad** [comment]
Or this:

![image](https://github.com/google/material-design-icons/assets/7026091/2dd91e6b-f9de-4e21-980f-892760c5f015)
